src/test/kotlin/com/atlassian/performance/tools/virtualusers/LoadTestIT.kt

```diff
@@ -96,34 +96,32 @@ class LoadTestIT {
     private fun <T> testWithJira(
         test: (URI) -> T
     ): T {
-        val docker = DockerClientBuilder.getInstance().build()
-        return docker
-            .createNetworkCmd()
-            .withName(UUID.randomUUID().toString())
-            .execAsResource(docker)
-            .use { network ->
-                val networkId = network.response.id
-                val dbImage = SudoSshUbuntuImage(docker, networkId, listOf(3306))
-                val jiraImage = SudoSshUbuntuImage(docker, networkId, listOf(8080))
-                dbImage.runInUbuntu { db ->
-                    jiraImage.runInUbuntu { jira ->
-                        test(runJiraServer(jira, db))
-                    }
-                }
+        fun exposePort(
+            port: Int
+        ) = Consumer { container: GenericContainer<*> ->
+            container.addExposedPort(port)
+        }
+
+        val dbContainer = SshUbuntuContainer(exposePort(3306))
+        val jiraContainer = SshUbuntuContainer(exposePort(8080))
+        return dbContainer.start().use { db ->
+            jiraContainer.start().use { jira ->
+                test(runJiraServer(jira, db))
             }
+        }
     }
 
     private fun runJiraServer(
-        jira: SudoSshUbuntuContainer,
-        db: SudoSshUbuntuContainer
+        jira: SshUbuntu,
+        db: SshUbuntu
     ): URI {
-        val publishedJiraPort = jira.ports.bindings[ExposedPort.tcp(8080)]!!.single().hostPortSpec.toInt()
+        val publishedJiraPort = jira.container.getMappedPort(8080)
         val jiraUri = URI("http://localhost:$publishedJiraPort/")
-        db.ssh.newConnection().use {
+        db.ssh.becomeUsable().newConnection().use {
             dataset.database.setup(it)
             dataset.database.start(jiraUri, it)
         }
-        startJiraNode(jira.ssh, db.peerIp)
+        startJiraNode(jira.ssh.becomeUsable(), db.container.getContainerIpAddress())
```

Comment:
https://circleci.com/gh/dagguh/virtual-users/60 : 
> Error while executing sudo add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu bionic

Hmm, the `CopyPastedDocker` fails to install on Ubuntu `bionic`. Before this PR we're using `xenial`.
Raised: https://ecosystem.atlassian.net/browse/JPERF-449

Reply:
Here's the problem.
[`ssh-ubuntu` is based on `rastasheep`](https://github.com/atlassian/ssh-ubuntu/blob/release-0.2.2/docker/Dockerfile#L1), which [only supports LTS Ubuntu versions](https://hub.docker.com/r/rastasheep/ubuntu-sshd/tags), e.g. 16, 18.

[`infrastructure` hardcodes Docker for Ubuntu 17](https://github.com/atlassian/infrastructure/blob/release-4.14.4/src/main/kotlin/com/atlassian/performance/tools/infrastructure/Docker.kt#L31), which is not LTS and therefore not available as rastasheep.

So we gotta unhardcode it from infra AKA https://ecosystem.atlassian.net/browse/JPERF-449 mentioned earlier.